src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,44 @@
+package lotto.controller;
+
+import lotto.domain.model.LottoMoney;
+import lotto.domain.model.LottoResult;
+import lotto.domain.model.LottoTicket;
+import lotto.domain.model.LottoTickets;
+import lotto.domain.service.LottoService;
+import lotto.view.InputView;
+import lotto.view.ResultView;
+
+public class LottoController {
+
+    private final LottoTickets userTickets;
+    private final LottoTicket winningTicket;
+    private final LottoMoney lottoMoney;
+
+    private LottoController(LottoTickets userTickets, LottoTicket winningTicket,
+            LottoMoney lottoMoney) {
```

Comment:
개행이 어색한 것 같아요.
차라리 이렇게 표현해보면 어떨까요?
```suggestion
    private LottoController(
        LottoTickets userTickets,
        LottoTicket winningTicket,
        LottoMoney lottoMoney
    ) {
```